packages/ruby/spec/readme/har_spec.rb

```diff
@@ -1,9 +1,103 @@
 require "readme/har"
+require "rack/lint"
 
 RSpec.describe Readme::Har do
-  it "returns a valid har object" do
-    har = Readme::Har.new(double)
+  describe "#to_json" do
+    it "builds the correct values out of the env" do
+      time = Time.now
+      allow(Time).to receive(:now).and_return(time)
```

Comment:
The verification around time here is slightly opaque/feels like it's not as front-and-center as maybe it should be?

Perhaps it makes more sense to split out a test specifically around that – but timing the server response seems like something that we'd want to be very explicit is correct, and with the assertion of some of the rest of this, it feels like it gets lost a bit.

Reply:
Good point. I think we want this object to be "pure" anyway so we'll do the actual time checking in the middleware and pass the before/after times into this object.